cmd/crictl/container.go

```diff
@@ -624,15 +624,11 @@ func ListContainers(client pb.RuntimeServiceClient, opts listOptions) error {
 		createdAt := time.Unix(0, c.CreatedAt)
 		ctm := units.HumanDuration(time.Now().UTC().Sub(createdAt)) + " ago"
 		if !opts.verbose {
-			id := c.Id
-			if !opts.noTrunc {
-				id = strings.TrimPrefix(c.Id, "")
-				if len(id) > truncatedIDLen {
-					id = id[:truncatedIDLen]
-				}
-			}
+			id := getTruncatedID(opts.noTrunc, c.Id, "")
```

Comment:
I think this should be:
```
if !opts.noTrunc {
  id := getTruncatedID(c.Id, "")
}
```

It is weird that function `getTruncatedID` has a bool parameter to indicate returning a non-truncated id.

Reply:
done